### ISSUE TITLE
Add default configuration file locations

### ISSUE THREAD

**Issue by scottschreckengaust** [zricethezav/gitleaks#698]
**Is your feature request related to a problem? Please describe.**
How to add extra rules if necessary with false-positives without scripting.

**Describe the solution you'd like**
If and when there are configuration files found relative to the `--path`,  automatically override without having to use the `--config-path` or `--additional-config` parameters. Propose`gitleaks.toml` or `gitleaks-additional.toml` respectively.

**Describe alternatives you've considered**
When running against several repositories, using the `--additional-config=gitleaks.toml` requires every repo to have those files, and causes an exception if missing:
> no such file or directory

**Additional context**
Ideally, the `gitleaks --path <working_directory>` would find configuration file(s) if they exist and apply them.

cc @zricethezav

**Comment by zricethezav** [comment]
@scottschreckengaust great suggestion. This will be added in the upcoming version 8.

```
  -c, --config string          config file path
                               order of precedence:
                               1. --config/-c
                               2. (--source/-s)/.gitleaks.toml
                               if --config/-c is not set and no .gitleaks.toml/gitleaks.toml present
                               then .gitleaks.toml will be written to (--source/-s)/.gitleaks.toml for future use
```

**Comment by zricethezav** [comment]
available in v8.0.0
